net/pfSense-pkg-freeradius2/Makefile

```diff
@@ -55,6 +58,10 @@ do-install:
 		${STAGEDIR}${PREFIX}/pkg
 	${INSTALL_DATA} ${FILESDIR}${DATADIR}/info.xml \
 		${STAGEDIR}${DATADIR}
+	${INSTALL_DATA} ${FILESDIR}${PREFIX}/etc/raddb/modules/googleauth \
+		${STAGEDIR}${PREFIX}/etc/raddb/modules
+	${INSTALL_SCRIPT} ${FILESDIR}${PREFIX}/etc/raddb/scripts/googleauth.py \
+		${STAGEDIR}${PREFIX}/etc/raddb/scripts
```

Comment:
Almost there. It's still missing `${MKDIR}` entries, like at line 31 in order to get `${STAGEDIR}/etc/raddb/modules` and `${STAGEDIR}/etc/raddb/scripts` directories created